openslides/agenda/static/js/agenda/base.js

```diff
@@ -66,6 +75,27 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
                                element.id == self.content_object.id;
                     };
                     return typeof _.findKey(projector.elements, predicate) === 'string';
+                },
+                // project list of speakers
+                projectListOfSpeakers: function() {
+                    return $http.post(
+                        '/rest/core/projector/1/prune_elements/',
+                        [{name: 'agenda/item', id: this.id, list_of_speakers: true}]
```

Comment:
Here I have the question if we want to project the list of speakers of an agenda item or directly of the related object. Example: `{"name": "motions/motion", "id": "5", "list_of_speakers": true}`